src/alfasim_sdk/_internal/constants.py

```diff
@@ -169,6 +169,21 @@ class SolidsModelType(Enum):
     FromPlugin = "from_plugin"
 
 
+class FrictionFactorCorrelationModelType(Enum):
+    """
+    Informs which friction factor correlation model should be used:
+
+    - ModelDefault:
+        From ALFAsim's internal default friction factor correlation.
+
+    - FromPlugin:
+        Friction factor correlation is calculated from plugin hooks with an external implementation.
+    """
+
+    ModelDefault = "from_package"
```

Comment:
This is a strange name, perhaps `internal` would make more sense?


```suggestion
    Internal = "internal"
```